manifests/s/Skrepysh/mineSquid/2.22/Skrepysh.mineSquid.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Skrepysh.mineSquid
+PackageVersion: "2.22"
+InstallerType: inno

```

Comment:
```suggestion
InstallerType: inno
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Reply:
i will include this file manually in my next release...